src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -37,7 +37,7 @@ public class PatchMapDrawer : Control
     /// <summary>
     ///   The representation of connections between regions, so we won't draw the same connection multiple times
     /// </summary>
-    private readonly Dictionary<Int2, Tuple<Vector2, Vector2, Vector2>> connections = new();
+    private readonly Dictionary<Int2, Vector2[]> connections = new();
 
```

Comment:
Here

Reply:
Alright

Reply:
Ill leave as is now. Its not that much anyway.